### PR TITLE
Readme formatting

### PR DESCRIPTION
**Pull request by kdb424**
Forgot the readme

### CODE REVIEW

File: boards/boardsource/Lulu/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[Boardsource](https://boardsource.xyz/store/5ec9df84c6b834480de6c3d0)  
+
+Extensions enabled by default  
+- [Layers](https://github.com/KMKfw/kmk_firmware/tree/master/docs/layers.md) Need more keys than switches? Use layers.
+- [RGB](https://github.com/KMKfw/kmk_firmware/tree/master/docs/rgb.md) Light it up
+- [BLE_Split](https://github.com/KMKfw/kmk_firmware/tree/master/docs/split.md) Connects halves without wires
+
+Common Extensions
+- [Split](https://github.com/KMKfw/kmk_firmware/tree/master/docs/split.md) Connects halves using a wire
+- [Power](https://github.com/KMKfw/kmk_firmware/tree/master/docs/power.md) Powersaving features for battery life
```
Comment (klardotsh) [inline boards/boardsource/Lulu/README.md:20]: These should all be relative links from the top of the readme (`/docs/split.md` for example)

**Review by klardotsh — APPROVED** [review]
:+1: modulo one bit of pedantry

File: boards/boardsource/Lulu/README.md (hunk 1 above)
Comment (kdb424) [reply boards/boardsource/Lulu/README.md:20]: Then I need to fix this several placed. Whoops.

**Comment by kdb424** [comment]
Changed readme formatting globally

**Pull request merged**